api/client/options.proto

```diff
@@ -187,4 +187,6 @@ message CommandLineOptions {
   repeated string labels = 28;
   // TransportSocket configuration to use in every request
   envoy.config.core.v3.TransportSocket transport_socket = 27;
+  // Do not perform the simple warmup call.
+  google.protobuf.BoolValue no_simple_warmup = 32;
```

Comment:
We have learned internally that naming boolean flags in the negative can end up being confusing. E.g:

--no-simple-warmup=False

Can we name the flag simple-warmup instead and set its default accordingly?

Reply:
fixed in 5940d5601eaa9924782c57be5a67895a7a1dfdd5